etc/afpd/unix.h

```diff
@@ -202,4 +210,8 @@ extern int setfilunixmode   (const struct vol *, struct path*, const mode_t);
 extern int setfilowner      (const struct vol *, const uid_t, const gid_t, struct path*);
 extern void accessmode      (const AFPObj *obj, const struct vol *, char *, struct maccess *, struct dir *, struct stat *);
 
+#ifdef AFS	
+    #define accessmode afsmode
+#endif
```

Comment:
I thought we got rid of all traces of Andrew File System... but I assume this is something else?